secrets/secrets.go

```diff
@@ -90,8 +91,17 @@ func (s *Secrets) Detect(secretsChannel chan reporting.Secret, item plugins.Item
 		Raw: item.Content,
 	}
 	for _, value := range s.detector.Detect(fragment) {
-		itemId := getItemId(item.ID)
-		secretsChannel <- reporting.Secret{ID: itemId, Source: item.ID, Description: value.Description, StartLine: value.StartLine, StartColumn: value.StartColumn, EndLine: value.EndLine, EndColumn: value.EndColumn, Value: value.Secret}
+		itemId := getFindingId(item, value)
+		secretsChannel <- reporting.Secret{
+			ID:          itemId,
+			Description: item.Description,
+			Type:        value.Description,
+			StartLine:   value.StartLine,
+			StartColumn: value.StartColumn,
+			EndLine:     value.EndLine,
+			EndColumn:   value.EndColumn,
+			Value:       value.Secret,
+		}
```

Comment:
`Source: item.ID` changed to `Description: item.Description`
`Description: value.Description` changed to `Type: value.Description`